source/isaaclab/isaaclab/actuators/actuator_base.py

```diff
@@ -85,8 +85,20 @@ class ActuatorBase(ABC):
     armature: torch.Tensor
     """The armature of the actuator joints. Shape is (num_envs, num_joints)."""
 
-    friction: torch.Tensor
-    """The joint friction of the actuator joints. Shape is (num_envs, num_joints)."""
+    max_actuator_velocity: torch.Tensor
```

Comment:
This warrants a proper documentation on what the new model is and how all these new parameters come into play.  Could we explain this better somewhere?

Also this breaks stuff with "friction" attribute?